### PR TITLE
ci: added github actions workflow

### PR DESCRIPTION
**Pull request by audrium**
Here is an example of GH actions workflow run: https://github.com/audrium/reana-workflow-monitor/actions/runs/343642776

### CODE REVIEW

**Review by ParthS007 — APPROVED** [review]
LGTM

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        with:
+          python-version: 2.7
+
+      - name: Install Python dependencies
+        run: |
+          pip install --upgrade pip setuptools py
+          pip install twine wheel
+          pip install -e .[all]
+
+      - name: Run pytest
```
Comment (mvidalgarcia) [inline .github/workflows/ci.yml:116]:
```suggestion
      - name: Run isort
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+          files: coverage.xml
+
+  docker-build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Build docker image
+        run: ./run-tests.sh --check-docker-build
```
Comment (mvidalgarcia) [inline .github/workflows/ci.yml]: Missing endline

File: README.rst [hunk 3]
```diff
@@ -2,14 +2,15 @@
  REANA-Workflow-Monitor
 ========================
 
-.. image:: https://img.shields.io/travis/reanahub/reana-workflow-monitor.svg
-         :target: https://travis-ci.org/reanahub/reana-workflow-monitor
+
+. image:: https://github.com/reanahub/reana-workflow-monitor/workflows/CI/badge.svg
```
Comment (mvidalgarcia) [inline README.rst]: ⚠️

File: run-tests.sh [hunk 4]
```diff
@@ -1,15 +1,76 @@
...
+check_script () {
+    shellcheck run-tests.sh
+}
+
+check_pydocstyle () {
+    pydocstyle reana_workflow_monitor
+}
+
+check_isort () {
+    isort -rc -c -df -- */*.py
```
Comment (mvidalgarcia) [inline run-tests.sh]: https://github.com/reanahub/reana-message-broker/pull/27#discussion_r516641109

**Review by mvidalgarcia — APPROVED** [review]

**Pull request merged**